### PR TITLE
Grammar fixes

### PR DESCRIPTION
**Pull request by Mehvix**
### Summary
Small grammar changes.

### Full changelog
Word and grammar changes intended to make files easier to read.

### CODE REVIEW

File: UPGRADE.md [hunk 1]
```diff
@@ -64,7 +64,7 @@ complete list of changes and new features.
 ##### Configuration
 
 - The `server_tokens` and `latency_tokens` configuration properties have been
-  removed. Instead, a new `headers` configuration properties replaces them.
+  removed. Instead, a new `headers` configuration properties replace them.
```
Comment (james-callahan) [inline UPGRADE.md]: I think the original was correct here.

File: kong.conf.default [hunk 2]
```diff
@@ -508,7 +508,7 @@
 #dns_no_sync = off               # If enabled, then upon a cache-miss every
                                  # request will trigger its own dns query.
                                  # When disabled multiple requests for the
-                                 # same name/type will be synchronised to a
+                                 # same name/type will be synchronized to a
```
Comment (james-callahan) [inline kong.conf.default]: american vs english spelling. I'm not sure what our preferred style here is; I'd err on the side of leaving as-is.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -221,12 +221,12 @@ logical units of work. You are free to propose a patch with one or many
 commits, as long as their atomicity is respected. This means that no unrelated
 changes should be included in a commit.
 
-For example: you are writing a patch to fix a bug, but in your endeavour, you
+For example: you are writing a patch to fix a bug, but in your endeavor, you
```
Comment (james-callahan) [inline CONTRIBUTING.md]: american vs english spelling. I'm not sure what our preferred style here is; I'd err on the side of leaving as-is.

File: README.md [hunk 4]
```diff
@@ -74,7 +74,7 @@ transformations, and more through plugins.
   updated across nodes.
 - **Scalability**: Distributed by nature, Kong scales horizontally by simply
   adding nodes.
-- **Performance**: Kong handles load with ease by scaling and using NGINX at
+- **Performance**: Kong handles loads with ease by scaling and using NGINX at
```
Comment (james-callahan) [inline README.md]: I think the singular here is correct?

**Comment by Mehvix** [comment]
I've reverted the grammar changes you've commented on. I'll close this PR and open another with a single commit.

**Comment by james-callahan** [comment]
No need to open a new PR, just force push to the same branch and github will update this PR.

**Pull request closed without merging**